<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JackChooseDialog</class>
 <widget class="QDialog" name="JackChooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>124</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jack suit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Choose Jack suit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="SuitRadioButton" name="heartsSuit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="suitDescription" stdset="0">
           <cstring>♥</cstring>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">suitGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="SuitRadioButton" name="diamondsSuit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suitDescription" stdset="0">
           <cstring>♦</cstring>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">suitGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="SuitRadioButton" name="clubsSuit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suitDescription" stdset="0">
           <cstring>♣</cstring>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">suitGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="SuitRadioButton" name="spadesSuit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suitDescription" stdset="0">
           <cstring>♠</cstring>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">suitGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SuitRadioButton</class>
   <extends>QRadioButton</extends>
   <header>suitradiobutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>heartsSuit</tabstop>
  <tabstop>diamondsSuit</tabstop>
  <tabstop>clubsSuit</tabstop>
  <tabstop>spadesSuit</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>JackChooseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>JackChooseDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="suitGroup"/>
 </buttongroups>
</ui>
